src/importexport/guitarpro/internal/importptb.cpp

```diff
@@ -1228,7 +1228,6 @@ Err PowerTab::read()
     readDataInstruments(song.track1);
     staffInc = int(song.track1.infos.size());
     lastStaffMap.clear();
-    readDataInstruments(song.track1);
```

Comment:
This is just a duplicate of line 1228

Reply:
Looks like it's used to fill lastStaffMap

Reply:
Hmm... So there is a bug then, because it adds an extra staff which not exist in the original file. I'll have a close look.
Thanks for pointed this out.

Reply:
You're right. It does fills the lastStaffMap. The thing with extra stuff present only for file I'm working with. Other files are fine. So the problem is with the file, not the code.